driverconfig/config.yaml

```diff
@@ -108,18 +109,17 @@ drivers:
         external-health-monitor: gcr.io/k8s-staging-sig-storage/csi-external-health-monitor-controller:v0.4.0
   - name: isilon
     configVersions:
-      - configVersion: v2.4.0
+      - configVersion: v2.5.0
         useDefaults: true
         supportedVersions:
           - version: v121
-          - version: v122
           - version: v123
           - version: v124
+          - version: v125
       - configVersion: v2.3.0
         useDefaults: false
         supportedVersions:
           - version: v121
-          - version: v122
```

Comment:
Why are we removing v122 for config version 2.3.0? Previous versions should be kept untouched. Please revert this change

Reply:
reverted back